diff -rU1 pypyodbc-1.3.6/PKG-INFO pypyodbc-1.3.5/PKG-INFO
--- pypyodbc-1.3.6/PKG-INFO	2021-11-02 08:19:20.000000000 +0300
+++ pypyodbc-1.3.5/PKG-INFO	2021-11-02 08:09:18.000000000 +0300
@@ -2,8 +2,10 @@
 Name: pypyodbc
-Version: 1.3.6
-Summary: PyPyODBC - A Pure Python ODBC module by ctypes
-Home-page: https://github.com/jiangwen365/pypyodbc
+Version: 1.3.5
 Author: jiangwen365
 Author-email: jiangwen365 at gmail com
-License: MIT
+Maintainer: jiangwen365
+Maintainer-email: jiangwen365 at gmail com
+Home-page: https://github.com/jiangwen365/pypyodbc
+Summary: PyPyODBC - A Pure Python ODBC module by ctypes
+License: License :: OSI Approved :: MIT License
 Description: 
@@ -38,7 +40,18 @@
         **Install**
-        - pip install pypyodbc
-Keywords: Python,Database,Interface,ODBC,PyPy
-Platform: UNKNOWN
-Classifier: License :: OSI Approved :: MIT License
-Classifier: Programming Language :: Python
+         - pip install pypyodbc
+Keywords: Python Database Interface  ODBC PyPy
 Classifier: Development Status :: 4 - Beta
+Classifier: Programming Language :: Python :: 2.4
+Classifier: Programming Language :: Python :: 2.5
+Classifier: Programming Language :: Python :: 2.6
+Classifier: Programming Language :: Python :: 2.7
+Classifier: Programming Language :: Python :: 3
+Classifier: Programming Language :: Python :: 3.0
+Classifier: Programming Language :: Python :: 3.1
+Classifier: Programming Language :: Python :: 3.2
+Classifier: Programming Language :: Python :: 3.3
+Classifier: Programming Language :: Python :: 3.4
+Classifier: Programming Language :: Python :: Implementation :: CPython
+Classifier: Programming Language :: Python :: Implementation :: IronPython
+Classifier: Programming Language :: Python :: Implementation :: PyPy
+
Only in pypyodbc-1.3.6: README.md
diff -rU1 pypyodbc-1.3.6/pypyodbc.egg-info/PKG-INFO pypyodbc-1.3.5/pypyodbc.egg-info/PKG-INFO
--- pypyodbc-1.3.6/pypyodbc.egg-info/PKG-INFO	2021-11-02 08:19:20.000000000 +0300
+++ pypyodbc-1.3.5/pypyodbc.egg-info/PKG-INFO	2021-11-02 08:09:18.000000000 +0300
@@ -2,8 +2,10 @@
 Name: pypyodbc
-Version: 1.3.6
-Summary: PyPyODBC - A Pure Python ODBC module by ctypes
-Home-page: https://github.com/jiangwen365/pypyodbc
+Version: 1.3.5
 Author: jiangwen365
 Author-email: jiangwen365 at gmail com
-License: MIT
+Maintainer: jiangwen365
+Maintainer-email: jiangwen365 at gmail com
+Home-page: https://github.com/jiangwen365/pypyodbc
+Summary: PyPyODBC - A Pure Python ODBC module by ctypes
+License: License :: OSI Approved :: MIT License
 Description: 
@@ -38,7 +40,18 @@
         **Install**
-        - pip install pypyodbc
-Keywords: Python,Database,Interface,ODBC,PyPy
-Platform: UNKNOWN
-Classifier: License :: OSI Approved :: MIT License
-Classifier: Programming Language :: Python
+         - pip install pypyodbc
+Keywords: Python Database Interface  ODBC PyPy
 Classifier: Development Status :: 4 - Beta
+Classifier: Programming Language :: Python :: 2.4
+Classifier: Programming Language :: Python :: 2.5
+Classifier: Programming Language :: Python :: 2.6
+Classifier: Programming Language :: Python :: 2.7
+Classifier: Programming Language :: Python :: 3
+Classifier: Programming Language :: Python :: 3.0
+Classifier: Programming Language :: Python :: 3.1
+Classifier: Programming Language :: Python :: 3.2
+Classifier: Programming Language :: Python :: 3.3
+Classifier: Programming Language :: Python :: 3.4
+Classifier: Programming Language :: Python :: Implementation :: CPython
+Classifier: Programming Language :: Python :: Implementation :: IronPython
+Classifier: Programming Language :: Python :: Implementation :: PyPy
+
diff -rU1 pypyodbc-1.3.6/pypyodbc.egg-info/SOURCES.txt pypyodbc-1.3.5/pypyodbc.egg-info/SOURCES.txt
--- pypyodbc-1.3.6/pypyodbc.egg-info/SOURCES.txt	2021-11-02 08:19:20.000000000 +0300
+++ pypyodbc-1.3.5/pypyodbc.egg-info/SOURCES.txt	2021-11-02 08:09:18.000000000 +0300
@@ -1,2 +1 @@
-README.md
 pypyodbc.py
diff -rU1 pypyodbc-1.3.6/pypyodbc.egg-info/requires.txt pypyodbc-1.3.5/pypyodbc.egg-info/requires.txt
--- pypyodbc-1.3.6/pypyodbc.egg-info/requires.txt	2021-11-02 08:19:20.000000000 +0300
+++ pypyodbc-1.3.5/pypyodbc.egg-info/requires.txt	2021-11-02 08:09:18.000000000 +0300
@@ -1 +1 @@
-setuptools
+setuptools
\ No newline at end of file
diff -rU1 pypyodbc-1.3.6/setup.cfg pypyodbc-1.3.5/setup.cfg
--- pypyodbc-1.3.6/setup.cfg	2021-11-02 08:19:20.000000000 +0300
+++ pypyodbc-1.3.5/setup.cfg	2021-11-02 08:09:18.000000000 +0300
@@ -3,2 +3,3 @@
 tag_date = 0
+tag_svn_revision = 0
 
diff -rU1 pypyodbc-1.3.6/setup.py pypyodbc-1.3.5/setup.py
--- pypyodbc-1.3.6/setup.py	2021-11-02 08:18:48.000000000 +0300
+++ pypyodbc-1.3.5/setup.py	2021-11-02 08:09:18.000000000 +0300
@@ -12,39 +12,11 @@
     name='pypyodbc',
-    version='1.3.6',
-    description='PyPyODBC - A Pure Python ODBC module by ctypes',
+    version='1.3.4',
+    description='A Pure Python ctypes ODBC module',
     author='jiangwen365',
-    author_email='jiangwen365 at gmail com',
+    author_email='jiangwen365@gmail.com',
     url='https://github.com/jiangwen365/pypyodbc',
     py_modules=['pypyodbc'],
-      long_description="""
-
-GitHub `Home Page <https://github.com/jiangwen365/pypyodbc/>`_
-
-
-Also check out the `Wiki <https://github.com/jiangwen365/pypyodbc/wiki>`_ and 
-`Version History <https://github.com/jiangwen365/pypyodbc/wiki/Version-History>`_
-
-**Features**
-
-
- - One pure Python script, runs on CPython / IronPython / PyPy , Version 2.4 / 2.5 / 2.6 / 2.7 / 3.2 / 3.3, Win / Linux , 32 / 64 bit.
-
- - Almost totally same usage as pyodbc ( can be seen as a re-implementation of pyodbc in pure Python ).
-
- - Simple - the whole module is implemented in a single python script with less than 3000 lines.
-
-
-    `A Hello World script of pypyodbc database programing <http://code.google.com/p/pypyodbc/wiki/A_HelloWorld_sample_to_access_mssql_with_python>`_
-
-    `Built-in Access MDB file creation and compression functions on Windows <https://github.com/jiangwen365/pypyodbc/wiki/Access-MDB-support>`_
-
-
-    `A DBI 2.0 SQLAlchemy enabler driver for IronPython <http://code.google.com/p/pypyodbc/wiki/Enable_SQLAlchemy_on_IronPython>`_
-
-    `A DBI 2.0 SQLAlchemy enabler driver for *PyPy* <https://code.google.com/p/pypyodbc/wiki/Enable_SQLAlchemy_on_PyPy>`_
-
-
-
-**Install**
-- pip install pypyodbc""",
+      long_description="""\
+      A Pure Python ctypes ODBC module compatible with PyPy and almost totally same usage as pyodbc
+      """,
       classifiers=[
